### ISSUE TITLE
Where is the gradient update operation in `train_inner()` function?

### ISSUE THREAD

**Issue by jc-bao** [jc-bao/norml-fork#1]
[Function link](https://github.com/jc-bao/norml-fork/blob/c65c52a0b5fec22585d063ca7010fccca60dd595/maml_rl.py#L627)

In the description, the author mentioned that it 'computes the advantage and fine-tuned policy'. To get fine-tune the policy, the paper said they use vanilla policy gradient, which needs a calculated gradient. 

However, in this function, I cannot locate any gradient calculation operation. 

For reference, in a similar function called [`finetune()`](https://github.com/jc-bao/norml-fork/blob/c65c52a0b5fec22585d063ca7010fccca60dd595/maml_rl.py#L715), they use `session.run()` to apply the gradient descent.
